docs/adding_tracks.rst

```diff
@@ -626,6 +626,9 @@ Template Language
 
 Rally uses `Jinja2 <http://jinja.pocoo.org/docs/dev/>`_ as a template language so you can use Jinja2 expressions in track files.
 
+.. note::
```

Comment:
Reading the ticket (and thinking about the maximum visibility and benefit) I think we want to have a small example besides an operation.

What if we place a second example inside the [put pipeline operation](https://esrally.readthedocs.io/en/stable/track.html#put-pipeline) doc page showing a similar example to what we have in the Elasticsearch docs for [handling failure in pipelines](https://www.elastic.co/guide/en/elasticsearch/reference/current/handling-failure-in-pipelines.html).
We could raise the attention with a note that "if you need to use Elasticsearch Mustache `{{ }}` syntax you must escape it properly" (and link the note you added in this PR), for example:

```
{
  "name": "define-ip-geocoder",
  "operation-type": "put-pipeline",
  "id": "ip-geocoder",
  "body": {
    "description": "Extracts location information from the client IP.",
    "processors": [
      {
        "geoip": {
          "field": "clientip",
          "properties": [
            "city_name",
            "country_iso_code",
            "country_name",
            "location"
          ]
        }
      }
    ],
    "on_failure": [
      {
        "set" : {
          "field" : "_index",
          {% raw %}
          "value" : "failed-{{ _index }}"
          {% endraw %}
        }
      }        
    ]
}
```

^^ For the sake of not repeating things we could even just show `"processors": [ ... ],` as used in the Elasticsearch docs page.

Reply:
@dliappis I decided to just add to the existing example, and added a link from the adding tracks page.